rmf_fleet_adapter_python/src/adapter.cpp

```diff
@@ -0,0 +1,176 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/functional.h>
+#include <pybind11/iostream.h>
+#include <pybind11/chrono.h>
+#include <pybind11/eigen.h>
+#include <pybind11/stl.h>
+
+#include <memory>
+
+#include "rmf_traffic_ros2/Time.hpp"
+#include "rmf_fleet_adapter/agv/Adapter.hpp"
+#include "rmf_fleet_adapter/agv/test/MockAdapter.hpp"
+#include "rmf_fleet_adapter_python/PyRobotCommandHandle.hpp"
+
+namespace py = pybind11;
+namespace agv = rmf_fleet_adapter::agv;
+
+void bind_types(py::module &);
+void bind_graph(py::module &);
+void bind_shapes(py::module &);
+void bind_vehicletraits(py::module &);
+void bind_plan(py::module &);
+void bind_tests(py::module &);
+
+PYBIND11_MODULE(rmf_adapter, m) {
+    bind_types(m);
+    bind_graph(m);
+    bind_shapes(m);
+    bind_vehicletraits(m);
+    bind_plan(m);
+    bind_tests(m);
+
+    // ROBOTCOMMAND HANDLE =====================================================
+    // Abstract class
+    py::class_<agv::RobotCommandHandle, PyRobotCommandHandle,
+               std::shared_ptr<agv::RobotCommandHandle> > \
```

Comment:
Can we remove backslashes in non-macro code, and replace it with the opening parenthesis? 

```
py::class_<agv::RobotCommandHandle,` PyRobotCommandHandle,
               std::shared_ptr<agv::RobotCommandHandle> > (
```

The parameters can start on the next line

Reply:
also apply it throughout all the files